spec/index.html

```diff
@@ -820,20 +820,36 @@ <h3><code>rdf:nil</code></h3>
 
 <section id="ch_ttrei">
   <h2>Triple-term based reification</h2>
-  <p>A triple term is a construct defined in the RDF 1.2 Concepts and Abstract 
-    Syntax specification [[RDF12-CONCEPTS]], referring to an RDF triple that 
-    is used as a value, specifically, in the object position of another triple. 
-    Importantly, a triple term does not imply assertion; that is, the triple it 
-    denotes is not necessarily asserted in the RDF graph. This enables the 
-    representation of metadata or commentary about triples that may or may not 
-    be part of the asserted graph content, including potentially contradictory 
-    relationships. For a triple term to be considered an asserted triple, it 
-    must also appear explicitly in an RDF graph as a top-level triple. Otherwise, 
-    it remains unasserted and serves primarily as a reference or subject of metadata.</p>
+  <p>A triple term is a construct defined in the RDF 1.2 Concepts and Abstract Syntax specification 
+    [[RDF12-CONCEPTS]], referring to an RDF triple that is used as a value, specifically, in the object 
+    position of another triple. This use is a reference to a proposition, and importantly, a triple term 
+    does not by itself imply assertion; that is, the triple it denotes is not necessarily asserted in 
```

Comment:
```suggestion
    does not by itself imply assertion; that is, the proposition it denotes is not necessarily a fact asserted by
```
a triple term does not denote a triple, so this sentence was wrong (even before this PR).
This suggested change fixes this, aligning with the terminology defined in RDF Semantics.